vendor/opencl_icd_loader/opencl_icd_loader.BUILD

```diff
@@ -61,8 +61,8 @@ cc_library(
     linkopts = opencl_icd_loader_lnks,
     linkstatic = 1,
     local_defines = select({
-        "@bazel_tools//src/conditions:windows": [],
-        "//conditions:default": ["HAVE___SECURE_GETENV"],
+        "@bazel_tools//src/conditions:linux_x86_64": ["HAVE_SECURE_GETENV"],
+        "//conditions:default": [],
```

Comment:
@earhart @LLITCHEV @kdobros I had to make this change to make things work correctly in my environment. WDYT?

Reply:
Yeah... This is very sensitive on the environment, but Linux has the SEC_ENV, so this should be fine.

Reply:
I think you should delete the entire `local_defines` block.

At least when I tried it: this condition wasn't working on our Linux builds.  I suspect it's because we use "linux_x86_64" for our CPU, instead of "k8", which is what the condition uses.

Also, when I tried using `HAVE_SECURE_GETENV`, it didn't work on our CI; we needed the double-underscore version.